components/tag-list/tag-list-item-mixin.js

```diff
@@ -27,6 +27,11 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 			 * @type {boolean}
 			 */
 			clearable: { type: Boolean },
+			/**
+			 * Acts as a unique identifier for the tag in dispatched events
+			 * @type {string}
+			 */
+			key: { type: String },
```

Comment:
Should this be moved to the `tag-list-item` component, and then mixin consumers can set up their own if needed?

Reply:
Based on the conversation above, leaving this here might help encourage proper data retrieval for mixin consumers too.

Reply:
Agreed. And per offline conversation, if we are going to prove this as part of the event detail in the mixin, then I think it makes sense for this `key` to be defined in the mixin.